packages/tables/src/Table.php

```diff
@@ -339,6 +339,23 @@ public function getRecordUrl(Model $record): ?string
         return $callback($record);
     }
 
+    public function isRecordBulkActionEnabled(Model $record): bool {
+        if (!$this->isSelectionEnabled()) {
+            return false;
+        }
+
+        /** @var TableComponent $livewire */
+        $livewire = $this->getLivewire();
+
+        $callback = invade($livewire)->getTableRecordBulkActionsEnabled();
```

Comment:
No need to invade if the method is public:

```suggestion
        $callback = $livewire->getTableRecordBulkActionsEnabled();
```